packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -225,6 +225,14 @@ class ZapierBaseCommand extends Command {
       .join('\n')
       .trim();
   }
+
+  _recordAnalytics() {
```

Comment:
this could be overkill, but I did this before finding the bug in `analytics.js` itself. In any case, it's nice to centralize the behavior.